src/components/BarChart/__stories__/PositiveNegative.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import { ComponentStory } from '@storybook/react'
+import { format } from 'date-fns'
+import BarChart from '..'
+import { barChartPositiveNegativeData } from './mockData'
+
+export const PositiveNegative: ComponentStory<typeof BarChart> = () => (
```

Comment:
Can you add props here to have more beautiful code snippet and remove those:

![Screenshot 2022-10-24 at 15 27 57](https://user-images.githubusercontent.com/15812968/197537149-157f4088-6c75-4276-98a3-8284e78148aa.png)

Reply:
Should be something like:

```ts
export const PositiveNegative: ComponentStory<typeof BarChart> = props => <BarChart {...props} otherProp={value} ... />
```